test/Passes/test-matchers.mlir

```diff
@@ -0,0 +1,252 @@
+// RUN: tpp-opt %s -mlir-disable-threading=true -pass-pipeline="builtin.module(func.func(test-structural-matchers))" -o /dev/null 2>&1 | FileCheck %s
```

Comment:
Why is `-mlir-disable-threading=true` needed?

Reply:
not messing up the printing. We would run in parallel all the functions.

Reply:
Oh right, probably coz you won't get outputs in correct order for filecheck.